share/ext/common.sh

```diff
@@ -48,7 +48,7 @@ if [ "Darwin" == "$(uname)" ]; then
 fi
 
 ANDROID_NDK_VERSION=25b
-ANDROID_NDK_ROOT=${DYNAMO_HOME}/ext/SDKs/android-ndk-r${ANDROID_NDK_VERSION}
+ANDROID_NDK_ROOT=${ANDROID_NDK_ROOT:-"${DYNAMO_HOME}/ext/SDKs/android-ndk-r${ANDROID_NDK_VERSION}"}
```

Comment:
`ANDROID_NDK_ROOT` is already set in the GitHub Action. If it is not set we fall back to the one in `DYNAMO_HOME`.

Although now that I look at it perhaps we should first check if `${DYNAMO_HOME}/ext/SDKs/android-ndk-r${ANDROID_NDK_VERSION` exists and if it doesn't we fall back to whatever is already set (if anything).